src/multiplex/server.rs

```diff
@@ -223,6 +234,8 @@ where
             }
 
             // also try to make progress on sending
+            #[cfg(feature = "tracing")]
+            tracing::trace!(finish = ?this.finish, "transport.poll_flush");
```

Comment:
`finish` is a bool, so it can be recorded as a typed value rather than using `fmt::Debug`:
```suggestion
            tracing::trace!(finish = this.finish, "transport.poll_flush");
```